src/olympia/addons/admin.py

```diff
@@ -194,10 +198,38 @@ def wrapper(*args, **kwargs):
         ]
         return custom_urlpatterns + urlpatterns
 
+    def authors_links(self, obj):
+        # Note: requires .transform(Addon.attach_all_authors) to have been
+        # applied to fill all_authors property and role on each user in it.
+        authors = obj.all_authors
+        return format_html(
+            '<ul>{}</ul>',
+            format_html_join(
+                '',
+                '<li><a href="{}">{} ({}{})</a></li>',
+                (
+                    (
+                        urljoin(
+                            settings.EXTERNAL_SITE_URL,
+                            reverse(
+                                'admin:users_userprofile_change',
+                                args=(author.pk,)
+                            ),
+                        ),
+                        author.email,
+                        dict(amo.AUTHOR_CHOICES_UNFILTERED)[author.role],
```

Comment:
`author.get_role_display()`?

Reply:
Oh, you can't do that because it's not an `AddonUser` instance, hmm.

Reply:
Yeah...